backend/modules/user/user.controller.ts

```diff
@@ -0,0 +1,215 @@
+import { User } from "../../entities/user";
+import { AppDataSource } from "../../database/dbConnect";
+const bcrypt = require("bcrypt");
+import { UserService } from "./UserService";
+const passport = require("passport");
+
+const userRepo = AppDataSource.getRepository(User);
+
+const userService = new UserService();
+
+export const login = async (req: any, res: any, next: any) => {
+  passport.authenticate(
+    "local",
+    (err: Error, user: User, info: { message: string }) => {
+      if (err) {
+        return next(err);
+      }
+      if (!user) {
+        return res.status(401).send({ message: info.message });
+      }
```

Comment:
add some spacing between code blocks for better readability

Reply:
for example 

```
if () {
...
}

if () {
...
}

return ...
```

Reply:
Noted